### PR TITLE
Display user first/last name in breakout room participants selector

### LINKED ISSUE #551: in Breakout rooms setting, allow to show Human names
hello, 

is it possible to see Human readable names with emails or just human names in breakout rooms feature? 

image

thanks in advance :)

### PR DESCRIPTION
**Pull request by armccoy**
Fixes #551

### CODE REVIEW

File: mod_form.php [hunk 1]
```diff
@@ -422,7 +422,7 @@ public function definition() {
         // Getting Course participants.
         $courseparticipants = [];
         foreach ($participants as $participant) {
-            $courseparticipants[] = ['participantid' => $participant->id, 'participantemail' => $participant->email];
+            $courseparticipants[] = ['participantid' => $participant->id, 'participantname' => $participant->firstname . ' ' . $participant->lastname];
```
Comment (jrchamp) [inline mod_form.php:425]: It seems that Moodle prefers that you use the fullname() function when you want to show the "firstname lastname". This allows for internationalization.
```suggestion
            $courseparticipants[] = [
                'participantid' => $participant->id,
                'participantname' => fullname($participant),
            ];
```

Alternatively, if the fullname() isn't unique enough, we could use the name-addr email address format:
```suggestion
            $courseparticipants[] = [
                'participantid' => $participant->id,
                'participantname' => fullname($participant) . ' <' . $participant->email . '>',
            ];
```

File: templates/breakoutrooms_rooms.mustache [hunk 2]
```diff
@@ -52,7 +52,7 @@
                 "courseparticipants": [
                     {
                         "participantid": "2",
-                        "participantemail": "p@example.com"
+                        "participantname": "John Doe",
```
Comment (jrchamp) [inline templates/breakoutrooms_rooms.mustache:55]: Unfortunately, stock JSON doesn't allow trailing commas.
```suggestion
                        "participantname": "John Doe"
```

**Review by jrchamp — COMMENTED** [review]
Thanks @armccoy! A couple suggestions that should resolve the automated checks and provide a better experience for some non-English users.

File: mod_form.php (hunk 1 above)
Comment (jrchamp) [reply mod_form.php:425]: As discussed in our meeting: @armccoy will test the name-addr email address format to make sure it works well in this context.

Comment (armccoy) [reply mod_form.php:425]: Just tested the settings layout with full name and email. No layout issues. Let's go with that suggestion.

**Review by jrchamp — APPROVED** [review]
Looks good! @armccoy confirmed that the email change looks good and works well.

**Pull request merged**